src/Element/EntityAutocompleteMultiple.php

```diff
@@ -0,0 +1,238 @@
+        [$class, 'processAjaxForm'],
+      ],
+    ];
+  }
+
+  /**
+   * Apply multiple value features for the entity autocomplete form element.
+   *
+   * @param array $element
+   *   The element.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   * @param array $complete_form
+   *   The original complete form array.
+   *
+   * @return array
+   *   The array of element.
+   */
+  public static function processEntityAutocompleteMultiple(array &$element, FormStateInterface $form_state, array &$complete_form): array {
+    unset($element['#type']);
+    $element['#tree'] = TRUE;
+    $element['#field_name'] = $field_name = end($element['#array_parents']);
+    $element['#default_value'] = $element['#default_value'] ?? [];
+    $parents = $element['#parents'];
+
+    $field_state = [];
+    if (static::getWidgetState($parents, $field_name, $form_state) === NULL) {
+      $max = count($element['#default_value']);
+      $field_state['items_count'] = $max;
+      static::setWidgetState($parents, $field_name, $form_state, $field_state);
+    }
+    else {
+      $max = static::getWidgetState($parents, $field_name, $form_state)['items_count'];
+    }
+
+    for ($i = 0; $i <= $max; $i++) {
+      $element[$i]['target'] = [
+        '#type' => 'entity_autocomplete',
+        '#target_type' => $element['#target_type'],
+        '#selection_handler' => $element['#selection_handler'],
+        '#selection_settings' => $element['#selection_settings'],
+        '#validate_reference' => FALSE,
+        '#maxlength' => $element['#maxlength'],
+        '#default_value' => $element['#default_value'][$i] ?? NULL,
+        '#size' => $element['#size'],
+        '#placeholder' => $element['#placeholder'],
+      ];
+      $element[$i]['_weight'] = [
+        '#type' => 'weight',
+        '#title' => t('Weight for row @number', ['@number' => $i + 1]),
+        '#title_display' => 'invisible',
+        '#delta' => 10,
+        '#default_value' => $i,
+        '#weight' => 100,
+      ];
+    }
+
+    $id_prefix = implode('-', $parents);
+    $wrapper_id = Html::getUniqueId($id_prefix . '-add-more-wrapper');
+    $element['#prefix'] = '<div id="' . $wrapper_id . '">';
+    $element['#suffix'] = '</div>';
+    $element['add_more'] = [
+      '#type' => 'submit',
+      '#name' => strtr($id_prefix, '-', '_') . '_add_more',
+      '#value' => $element['#add_more_title'],
+      '#attributes' => ['class' => ['field-add-more-submit']],
+      '#limit_validation_errors' => [$element['#array_parents']],
+      '#submit' => [[get_called_class(), 'addMoreSubmit']],
+      '#ajax' => [
+        'callback' => [get_called_class(), 'addMoreAjax'],
+        'wrapper' => $wrapper_id,
+        'effect' => 'fade',
+      ],
+    ];
+    // Remove garbage elements from root level of the element.
+    unset($element['#maxlength']);
+    return $element;
+  }
+
+  /**
+   * Element validator for checking.
```

Comment:
```suggestion
   * Validate autocomplete field values.
```
Always start a method description with a verb, it will help you formulate the sentence properly.

Reply:
Fixed